contracts/RewardsPool.sol

```diff
@@ -149,13 +149,16 @@ contract RewardsPool is IRewardsPool, Initializable, OwnableUpgradeable, Constan
             eraPool.unclaimDeployment += 1;
         }
 
+        uint256 laborAdd = amount;
         // indexer joined the deployment pool firstly.
         if (pool.labor[indexer] == 0) {
             IStakingManager stakingManager = IStakingManager(settings.getStakingManager());
             uint256 myStake = stakingManager.getTotalStakingAmount(indexer);
-            require(myStake > 0, 'RP003');
```

Comment:
allow myStake = 0, otherwise channel termination will fail forever and consumer can not get the remnant token back.